redpipe-engine/src/main/java/net/redpipe/engine/swagger/RxModelConverter.java

```diff
@@ -0,0 +1,52 @@
+package net.redpipe.engine.swagger;
+
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import io.swagger.converter.ModelConverter;
+import io.swagger.converter.ModelConverterContext;
+import io.swagger.models.Model;
+import io.swagger.models.properties.Property;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class RxModelConverter implements ModelConverter {
+
+    private static final List<String> delegateToFirstTypeArg = new ArrayList<>();
+    static {
+        delegateToFirstTypeArg.add(Single.class.getName());
+        delegateToFirstTypeArg.add(Observable.class.getName());
+        delegateToFirstTypeArg.add(rx.Single.class.getName());
+        delegateToFirstTypeArg.add(rx.Observable.class.getName());
+    }
+
+
+    @Override
+    public Property resolveProperty(Type type, ModelConverterContext context, Annotation[] annotations, Iterator<ModelConverter> chain) {
+        Type delegateType = type;
+        if (type instanceof ParameterizedType) {
+            ParameterizedType param = (ParameterizedType) type;
+            if (delegateToFirstTypeArg.contains(param.getRawType().getTypeName())) {
```

Comment:
Shouldn't this be `.startsWith()`?

Reply:
It's a `List<String>` containing the names of the ignored types. It should be a perfect match, no ?
I can do a `find` instead of `list.contains` but not sure it's interesting.

Reply:
Ah, I read that wrong, excuse me you're right :)